distributor-api-v1/operations.md

```diff
@@ -331,6 +338,7 @@ Negative or zero end offsets of the daily time unit define the daily service as
 | `AdditionalLegalStatements` | array of [Localized text](./operations.md#localized-text) | required | Additional legal statements. |
```

Comment:
ImageIds is still not removed from `hotels/get`

Reply:
I'll remove it in followup

Reply:
https://github.com/MewsSystems/gitbook-distributor-guide/commit/37a0258628f52f1e08b110714ae27b8508b0bbf2